electron/src/locale/locale.ts

```diff
@@ -219,6 +222,7 @@ export const SUPPORTED_LANGUAGES = {
   pt: 'Português do Brasil',
   ro: 'Română',
   ru: 'Русский',
+  si: 'සිංහල',
```

Comment:
@Rru247 is this correct?

Reply:
@ffflorian yes. (sorry didnt check inbox).